src/modules/src/controller_indi.c

```diff
@@ -31,15 +31,20 @@ static float bound_control_input = 32000.0f;
 static attitude_t attitudeDesired;
 static attitude_t rateDesired;
 static float actuatorThrust;
-static float roll_kp = 6.0f;
-static float pitch_kp = 6.0f;
-static float yaw_kp = 6.0f;
+static float roll_kp = 5.0f;
+static float pitch_kp = 5.0f;
+static float yaw_kp = 5.0f;
+
+static float attYawError; 
 
 static float r_roll;
 static float r_pitch;
 static float r_yaw;
 static float accelz;
 
+static vector_t refOuterINDI;		// Reference values from outer loop INDI
+static float outerLoopActive; 		// if 1, outer loop INDI is activated
```

Comment:
OuterloopActive is a boolean right? You should define this as a boolean and initialize it as = false. (don't forget to change this at the parameters as well)

Reply:
I've changed it as you suggested, however I am not sure if there is a boolean type for parameters (that's why I defined it as `PARAM_UINT8`).

Reply:
Indeed, there is no boolean for parameters, but uint8 is just fine. Thanks for changing it